src/instructlab/model/accelerated_train.py

```diff
@@ -77,17 +82,29 @@ def accelerated_train(
             pprint.pformat(train_args),
         )
 
-        if not (phased_phase1_data and phased_phase2_data):
-            # pylint: disable=broad-exception-raised
-            raise Exception(
-                "End-to-end training minimally requires: `--phased-phase1-data`, and `--phased-phase2-data`. One or more wasn't correctly specified."
-            )
-
-        # if they both exist, must be Path objects
-        if not (phased_phase1_data.exists() and phased_phase2_data.exists()):
-            raise FileNotFoundError(
-                "Data for both phase1 and phase2 must be specified for phased training."
-            )
+        if strategy == SupportedTrainingStrategies.LAB_MULTIPHASE.value:
+            if not (phased_phase1_data and phased_phase2_data):
+                # pylint: disable=broad-exception-raised
+                raise Exception(
+                    f"{SupportedTrainingStrategies.LAB_MULTIPHASE} training minimally requires: `--phased-phase1-data`, and `--phased-phase2-data`. One or more wasn't correctly specified."
+                )
+
+            # if they both exist, must be Path objects
+            if not (phased_phase1_data.exists() and phased_phase2_data.exists()):
+                raise FileNotFoundError(
+                    "Data for both phase1 and phase2 must be specified for phased training."
+                )
+        else:  # strategy == SupportedTrainingStrategies.LAB_SKILLS_ONLY.value
+            if not phased_phase2_data:
+                # pylint: disable=broad-exception-raised
+                raise Exception(
+                    f"{SupportedTrainingStrategies.LAB_SKILLS_ONLY} training minimally requires: `--phased-phase2-data`"
+                )
+
+            if not phased_phase2_data.exists():
+                raise FileNotFoundError(
+                    "Data for both phase1 and phase2 must be specified for phased training."
```

Comment:
Why mention phase 1 here if you're not checking for it and it's not needed for skills only training?

Reply:
Thanks, I missed updating this message.  Fixed now.